libsel4vm/src/arch/arm/vgic/vdist.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+//#define DEBUG_IRQ
+//#define DEBUG_DIST
+
+#ifdef DEBUG_IRQ
+#define DIRQ(...) do{ printf("VDIST: "); printf(__VA_ARGS__); }while(0)
+#else
+#define DIRQ(...) do{}while(0)
+#endif
+
+#ifdef DEBUG_DIST
+#define DDIST(...) do{ printf("VDIST: "); printf(__VA_ARGS__); }while(0)
+#else
+#define DDIST(...) do{}while(0)
+#endif
+
+#define IRQ_IDX(irq) ((irq) / 32)
+#define IRQ_BIT(irq) (1U << ((irq) % 32))
+
+extern const struct vgic_dist_device dev_vgic_dist;
```

Comment:
if we put this here, for consistency we should also include a header that has the type definition.

Reply:
This doesn't need to be here anyway.